src/astro/databases/snowflake.py

```diff
@@ -20,6 +35,15 @@ class SnowflakeDatabase(BaseDatabase):
     """
 
     def __init__(self, conn_id: str = DEFAULT_CONN_ID):
+        self.native_support: Dict[Any, str] = {FileLocation.GS: "gs_to_snowflake"}
```

Comment:
@sunank200  We should define native_support and  optimized_path_supported_file_types as static variables. Also, can we be consistent with [PR](https://github.com/astronomer/astro-sdk/pull/489) for the names of the variables?